framework/src/main/java/org/checkerframework/common/wholeprograminference/scenelib/ASceneWrapper.java

```diff
@@ -52,7 +52,7 @@
     private final AScene theScene;
 
     /** The classes in the scene. */
-    private final Map<@BinaryName String, AClassWrapper> classes = new HashMap<>();
+    private final Map<@BinaryName String, AClass> classes = new HashMap<>();
```

Comment:
You should be able to remove this field and replace all uses of it with the underlying `classes` field of the `AScene`, because each `AClass` should now have everything necessary.

Reply:
Thanks.  That was a significant oversight.